tests/sbc_test.py

```diff
@@ -76,6 +76,80 @@ def simulator(theta):
     get_nltp(thetas, xs, posterior)
 
 
+@pytest.mark.slow
+@pytest.mark.parametrize("prior", ("boxuniform", "independent"))
+@pytest.mark.parametrize("method, sampler", [(SNLE, "vi")])
+def test_consistent_sbc_results(method, prior, sampler, model="mdn"):
+    """Tests running inference and then SBC and obtaining nltp."""
+
+    num_dim = 2
+    if prior == "boxuniform":
+        prior = BoxUniform(-torch.ones(num_dim), torch.ones(num_dim))
+    else:
+        prior = MultipleIndependent([
+            Uniform(-torch.ones(1), torch.ones(1)) for _ in range(num_dim)
+        ])
+
+    num_simulations = 200
```

Comment:
```suggestion
    num_simulations = 1000
```
we need to estimator to be somewhat accurate to get consistent SBC results.

Reply:
see above about my point of NPE vs SNLE.

Reply:
addressed with 89e1ffd1